coding/simple_dense_coding.cpp

```diff
@@ -60,7 +60,11 @@ SimpleDenseCoding::SimpleDenseCoding(SimpleDenseCoding && rhs)
 
 uint8_t SimpleDenseCoding::Get(uint64_t i) const
 {
-  ASSERT_LESS(i, Size(), ());
+  // i can be greater than Size() for features created by user in the Editor.
```

Comment:
No, this class is too basic. Please, move that chec to users of this class, like RankTablr.